packages/types/src/block-kit/block-elements.ts

```diff
@@ -1,65 +1,149 @@
+   */
   initial_date?: string;
 }
 
+// TODO: breaking change: `Action` the datetimepicker extends from has an optional action_id, but is required for
+// datetimepicker.
 /**
- * @description An element that allows the selection of a time of day formatted as a UNIX timestamp. On desktop
+ * @description Allows users to select both a date and a time of day, formatted as a Unix timestamp. On desktop
  * clients, this time picker will take the form of a dropdown list and the date picker will take the form of a dropdown
  * calendar. Both options will have free-text entry for precise choices. On mobile clients, the time picker and date
  * picker will use native UIs.
- * {@link https://api.slack.com/reference/block-kit/block-elements#datetimepicker}
+ * @see {@link https://api.slack.com/reference/block-kit/block-elements#datetimepicker Datetime picker element reference}.
+ * @see {@link https://api.slack.com/interactivity/handling This is an interactive component - see our guide to enabling interactivity}.
  */
 export interface DateTimepicker extends Action, Confirmable, Focusable {
+  /**
+   * @description The type of element. In this case `type` is always `datetimepicker`.
+   */
   type: 'datetimepicker';
   /**
    * @description The initial date and time that is selected when the element is loaded, represented as a UNIX
-   * timestamp in seconds. This should be in the format of 10 digits, for example 1628633820 represents the date and
+   * timestamp in seconds. This should be in the format of 10 digits, for example `1628633820` represents the date and
    * time August 10th, 2021 at 03:17pm PST.
    */
   initial_date_time?: number;
 }
 
+// TODO: breaking change: `Action` the email extends from has an optional action_id, but is required for email inputs.
 /**
- * @description An email input element, similar to the {@see PlainTextInput} element, creates a single line field where
- * a user can enter an email address.
- * {@link https://api.slack.com/reference/block-kit/block-elements#email}
+ * @description Allows user to enter an email into a single-line field.
+ * @see {@link https://api.slack.com/reference/block-kit/block-elements#email Email input element reference}.
+ * @see {@link https://api.slack.com/interactivity/handling This is an interactive component - see our guide to enabling interactivity}.
  */
 export interface EmailInput extends Action, Dispatchable, Focusable, Placeholdable {
+  /**
+   * @description The type of element. In this case `type` is always `email_text_input`.
+   */
   type: 'email_text_input';
   /**
    * @description The initial value in the email input when it is loaded.
    */
   initial_value?: string;
 }
 
+/**
+ * @description Displays an image as part of a larger block of content. Use this `image` block if you want a block with
+ * only an image in it.
+ * @see {@link https://api.slack.com/reference/block-kit/block-elements#image Image element reference}.
+ */
 export interface ImageElement {
```

Comment:
Could take a similar approach later of aliasing `ImageElement` -> `Image` as proposed for ConfirmationDialog down below with the motivation of lining up the terms between types and api.slack.com

Reply:
I considered this as well but there's, confusingly, both an image block as well as an image block element ...